crates/eip4844/src/cryptography.rs

```diff
@@ -0,0 +1,478 @@
+use bls12_381::{ff::PrimeField, reduce_bytes_to_scalar_bias, G1Point, Scalar};
+use sha2::{Digest, Sha256};
+
+use crate::{BlobRef, KZGCommitment};
```

Comment:
Ideally we get rid of this too, since `Blob` , `KZGCommitment` and `KZGProof` are types on the 4844 crate. The cryptography crate should ideally only know about Scalars, polynomials (slice of scalars) and G1Point.

This does incur extra serialization costs, but its negligible and I think the separation of the cryptography from the 4844 code is much more desirable

Reply:
I think both of them are for Fiat-Shamir and it seems a bit specific to 4844, perhaps we could also move them out of cryptography module?